### ISSUE TITLE
[DOXIA-399] Links with '_' and '-' don't work

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-doxia#392]
**[Nathan Sowatskey](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=nsowatsk@cisco.com)** opened **[DOXIA-399](https://issues.apache.org/jira/browse/DOXIA-399?redirect=false)** and commented

See links.confluence in the attached project.

Neither the link to PAC or WPAD are rendered properly.

---

**Affects:** 1.1.3

**Attachments:**
- [toc-test-case.zip](https://issues.apache.org/jira/secure/attachment/12715605/toc-test-case.zip) (_4.25 kB_)
- [toc-test-case.zip](https://issues.apache.org/jira/secure/attachment/12715395/toc-test-case.zip) (_4.25 kB_)
- [toc-test-case.zip](https://issues.apache.org/jira/secure/attachment/12715424/toc-test-case.zip) (_4.22 kB_)

**Comment by jira-importer** [comment]
**[Nathan Sowatskey](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=nsowatsk@cisco.com)** commented

Fixed/updated test case showing slight fix using \.

**Comment by jira-importer** [comment]
**[Nathan Sowatskey](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=nsowatsk@cisco.com)** commented

Workaround with '\' before all '_'s.

**Comment by jira-importer** [comment]
**[Nathan Sowatskey](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=nsowatsk@cisco.com)** commented

Use '\_' in links. For example:

[PAC](http://en.wikipedia.org/wiki/Proxy\_auto-config)
[WPAD](http://en.wikipedia.org/wiki/Web\_Proxy\_Autodiscovery\_Protocol)

**Comment by jira-importer** [comment]
**[Lukas Theussl](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ltheussl)** commented

Has this been committed to svn? Only committers should close issues as fixed after the fix has been reviewed and comitted.

**Comment by jira-importer** [comment]
**[Nathan Sowatskey](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=nsowatsk@cisco.com)** commented

I provided a workaround in the attached project. Since there is a workaround, this is not a blocker. This Jira doesn't seem to let me change the priority, so I resolved it. Anyone looking for the same bug will see the workaround. If you want to re-open this for a code change, that's good too, but your choice :-)
